### DISCUSSION TITLE
Files with signatures taking long time to convert

### DISCUSSION THREAD

**Discussion by Anki0804** [gotenberg/gotenberg#456] [Q&A]
Hi All

My java code is calling the gotenberg API through HttpClient post method to convert files to pdf.
It is taking around 6-8 minutes, to convert doc files with signatures to PDF .
Tried using version 7.4.2 and 7.4.3 both.
Attached logs from docker container for gotenberg.
Is there any specific setting which can improve the performance in this case?
<img width="848" alt="Gotenberg docker logs" src="https://user-images.githubusercontent.com/104967074/166929274-e3e40f3a-e9de-4148-a608-fbe15098e7d4.PNG">

**Comment by gulien** [comment]
Hello @Anki0804,

You could try version `7.5.2`, even if I don't think it will change anything.

Are your documents heavy?

**Reply by Anki0804** [reply]
No files are not huge but they have signatures.

**Reply by gulien** [reply]
Ok, interesting 🧐
